charts/velocity-operator/templates/manager-rbac.yaml

```diff
@@ -1,17 +1,11 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: {{ include "chart.fullname" . }}-manager-role
+  name: {{ include "chart.fullname" . }}-manager-role{{ .Values.globalSuffix | default "" }}
   labels:
   {{- include "chart.labels" . | nindent 4 }}
 rules:
-- apiGroups:
-  - "" # core api group
-  - apps
```

Comment:
Will siso approve this?

Reply:
Apparently, we got a lot of problems and a few customers told us it's totally fine. We'll add hardening to that when we get requests for that 🙏🏼

Reply:
I've added a flag for that